config/noi.json

```diff
@@ -0,0 +1,25 @@
+{
+    "feed_data": {
+        "system_information": {
+            "email": "info@carsharing.bz.it",
+            "feed_contact_email": "mail@leonard.io",
+            "language": "de",
+            "license_url": "https://www.govdata.de/dl-de/by-2-0",
+            "license_id": "DL-DE-BY-2.0",
+            "attribution_organization_name": "Konsortialgenossenschaft Car Sharing Südtirol Alto Adige",
+            "attribution_url": "https://www.carsharing.bz.it",
+            "name": "Carsharing Südtirol Alto Adige",
+            "operator": "Konsortialgenossenschaft Car Sharing Südtirol Alto Adige",
+            "phone_number": "+39 0471 061319",
+            "privacy_url": "https://www.carsharing.bz.it/de/privacy/",
+            "purchase_url": "https://www.carsharing.bz.it/",
+            "system_id": "carsharing-suedtirol-alto-adige",
+            "terms_last_updated": "2022-02-15",
+            "terms_url": "https://www.carsharing.bz.it/de/impressum/",
+            "timezone": "Europe/Rome",
+            "url": "https://www.carsharing.bz.it/de/"
+        }
+    },
+    "system_id": "10024",
+    "provider_id": 84
```

Comment:
These are required for Cantamen based providers, not this one, I suppose

Reply:
93f498f34d5db56981d811aef90a3525bd0c694c